### PR TITLE
[dv] Add rv_core_ibex chip level tests to testplan

### PR DESCRIPTION
**Pull request by GregAC**
Signed-off-by: Greg Chadwick <gac@lowrisc.org>

### CODE REVIEW

File: hw/top_earlgrey/data/chip_testplan.hjson [hunk 1]
```diff
@@ -2587,6 +2587,42 @@
...
+    //////////////////
+    // CPU          //
+    // RV_CORE_IBEX //
+    //////////////////
+
+    {
+      name: chip_sw_rv_core_ibex_rnd
+      desc: '''Verify the functionality of the random number generation CSRs.
+
+               - Perform multiple reads from `RND_DATA` polling `RND_STATUS` in
```
Comment (sriyerg) [inline hw/top_earlgrey/data/chip_testplan.hjson:2599]: A minor detail missing here is to turn on the entropy ss.

**Review by sriyerg — APPROVED** [review]
LGTM with a note on making the steps a bit more detailed.

File: hw/top_earlgrey/data/chip_testplan.hjson [hunk 2]
```diff
@@ -2587,6 +2587,42 @@
...
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sw_rv_core_ibex_address_translation
+      desc: '''Verify the simple address translation functionality.
+
+               - Setup address translation for both I and D side and check
+                 correct translation for I and D accesses.
```
Comment (doreis-lowrisc) [inline hw/top_earlgrey/data/chip_testplan.hjson:2615]: There's two slots for I and D, I think we should mention both here to be tested.
Maybe we should have a step to test the feature:
`If a transaction matches multiple regions, the lowest indexed region has priority.`

**Review by doreis-lowrisc — APPROVED** [review]

**Comment by doreis-lowrisc** [comment]
LGTM

**Pull request merged**